README.md

```diff
@@ -227,6 +228,7 @@ If you have suggestions for new resources, improvements to methodologies, or cor
 | 2025 | CVPR | Prometheus | [Prometheus: 3D-Aware Latent Diffusion Models for Feed-Forward Text-to-3D Scene Generation](https://arxiv.org/abs/2412.21117) | [![link](https://img.shields.io/badge/Website-9cf)](https://freemty.github.io/project-prometheus/) | [![GitHub](https://img.shields.io/github/stars/XDimLab/Prometheus)](https://github.com/XDimLab/Prometheus) |
 | 2025 | ICLR | DynamicCity | [DynamicCity: Large-Scale Occupancy Generation from Dynamic Scenes](https://arxiv.org/abs/2410.18084) | [![link](https://img.shields.io/badge/Website-9cf)](https://dynamic-city.github.io/) | [![GitHub](https://img.shields.io/github/stars/3DTopia/DynamicCity)](https://github.com/3DTopia/DynamicCity) |
 | 2025 | arXiv | NuiScene | [NuiScene: Exploring Efficient Generation of Unbounded Outdoor Scenes](https://arxiv.org/abs/2503.16375) | [![link](https://img.shields.io/badge/Website-9cf)](https://3dlg-hcvc.github.io/NuiScene/) | [![GitHub](https://img.shields.io/github/stars/3dlg-hcvc/NuiScene?style=social)](https://github.com/3dlg-hcvc/NuiScene) |
+| 2025 | CVMJ | LPA3D | [LPA3D: 3D Room-Level Scene Generation from In-the-Wild Images](https://arxiv.org/abs/2504.02337) | | |
```

Comment:
LPA3D does not meet the following criteria:
1. It does not provide any project pages or source code.
2. CVMJ is not in the top-tier journal list.